cmd/plugins.go

```diff
@@ -355,7 +376,20 @@ func hoverPluginGet(dryRun bool) bool {
 					log.Errorf("Failed to resolve absolute path for plugin '%s': %v", dep.name, err)
 					os.Exit(1)
 				}
-				fileutils.AddLineToFile(filepath.Join(build.BuildPath, "go.mod"), fmt.Sprintf("replace %s => %s", pluginImportStr, path))
+
+				// mutation we're applying to go.mod
+				m := func(m *modfile.File) error {
```

Comment:
The assigned `m` is immediately "shadowed" in the function literal. The outer `m` should have a descriptive name.